asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/HttpRequestFactory.java

```diff
@@ -184,6 +183,14 @@ HttpGet getAttachmentByFileNameRequest(String contentId, String attachmentFileNa
         return getAttachmentByFileNameRequest;
     }
 
+    private static URIBuilder createUriBuilder(String path) {
+        try {
+            return new URIBuilder(path);
+        } catch (final URISyntaxException e) {
```

Comment:
Minor: we usually don't use the `final` modifier for local variables and parameters

Reply:
@cstettler I overlooked that, thanks for the hint. Fixed.